File: Competitive Programming 3/Section 1.4/Time/893 - Y3K Problem/Python/main.py
from calendar import calendar

def main():
    while 1:
        delta,d,m,y = map(int,input().split())
        if delta==0:
            break
        ans=calendar(y,m,d)+tim(days=delta)
        print(ans.day,ans.month,ans.year)
main()

File: Competitive Programming 3/Section 1.4/Time/893 - Y3K Problem/Java/Main.java
import java.util.*;

public class Main{
    static Scanner sc = new Scanner(System.in);
    public static void main(String[]args){
        int delta;
        while((delta=sc.nextInt())!=0){
            int d=sc.nextInt(),m=sc.nextInt(),y=sc.nextInt();
            GregorianCalendar gc = new GregorianCalendar(y, m - 1, d);
            gc.add(Calendar.DATE, delta);
            System.out.println(gc.get(Calendar.DATE) + " " + Integer.valueOf(gc.get(Calendar.MONTH) + 1) + " "
            + gc.get(Calendar.YEAR));
        }
    }
}